### DISCUSSION TITLE
How to connect to steam servers

### DISCUSSION THREAD

**Discussion by YamamotoBaconGuedt** [WilliamVenner/gmpublisher#194] [Help]
Im actually new to gmplublisher, when i downloaded the gmplublisher I noticed that I'm not connected to steam servers i tried to click the photo which is the "?" icon and it does not work. How can I fix this? Also by the time currently, it says it's still loading but it does not do anything.

**Comment by WilliamVenner** [comment]
You need to open Steam!
